src/components/BlenderLogbook/NewBlenderFillingEvent.tsx

```diff
@@ -160,7 +167,7 @@ export const NewBlenderFillingEvent: React.FC<
             <Form>
               <div className="fillingEventFlexRow">
                 <BasicInfoTile
-                  divingCylinderSets={divingCylinders}
+                  divingCylinderSets={cylinderSets}
```

Comment:
Voitais käyttää kaikkialla taatusti samaa nimeämistä eli tääkin olis `divingCylinderSets`. Koitin näitä jo vähän korvailla muuallekin